snowplow-demo-compose/src/main/java/com/snowplowanalytics/snowplowdemocompose/data/Tracking.kt

```diff
@@ -20,7 +20,7 @@ object Tracking {
     @Composable
     fun setup(namespace: String) : TrackerController {
         // Replace this collector endpoint with your own
-        val networkConfig = NetworkConfiguration("https://23a6-82-26-43-253.ngrok.io", HttpMethod.POST)
+        val networkConfig = NetworkConfiguration("http://192.168.0.20:9090", HttpMethod.POST)
```

Comment:
This should probably be a placeholder [like we have in the RN tracker](https://github.com/snowplow/snowplow-react-native-tracker/blob/master/example/src/App.js#L34) as everyone will need to adjust to their own external IP.